connector/src/main/java/org/geysermc/connector/network/translators/item/ItemRegistry.java

```diff
@@ -150,6 +152,89 @@ public static void init() {
             }
         }
 
+        Object2IntMap<String> bedrockBlockIdOverrides = new Object2IntOpenHashMap<>();
+        Set<String> blacklistedIdentifiers = new ObjectOpenHashSet<>();
+
+        // Load creative items
+        // We load this before item mappings to get overridden block runtime ID mappings
+        stream = FileUtils.getResource("bedrock/creative_items.json");
+
+        JsonNode creativeItemEntries;
+        try {
+            creativeItemEntries = GeyserConnector.JSON_MAPPER.readTree(stream).get("items");
+        } catch (Exception e) {
+            throw new AssertionError(LanguageUtils.getLocaleStringLog("geyser.toolbox.fail.creative"), e);
+        }
+
+        int netId = 1;
+        List<ItemData> creativeItems = new ArrayList<>();
+        JsonNode damageNode;
+        JsonNode countNode;
+        JsonNode blockRuntimeIdNode;
+        JsonNode nbtNode;
```

Comment:
Shouldn't these be moved into the loop so they are disposed of when the loop finishes by the JVM.

Reply:
Not sure if defining a new variable is better or reusing the same ones - I'll do some research now.

Reply:
It's in startup, so it doesn't really matter. I'll revert this for easier readability.